### ISSUE TITLE
[A11Y]: Accessibility update to Dropdown Button

### ISSUE THREAD

**Issue by LeeMellon** [phase2/outline#334]
**Describe the accessiblity issue**
`z-index should remain close to 0, definitely not 50.
aria-hidden should never be used in focusable items or focusable children (so shouldn't be in dropdown_panel), but is fine to use decorative icons 
dropdown button is missing aria-controls="..." and toggle aria-expanded="true"
aria-labelledby="dropdown" I think it can be removed, no?` As Per: @shaal 

Add/Update IDs for accessibility purposes and to prevent multiple elements in the light DOM having the same ID.
